easyfinance.client/src/app/features/project/detail-project/detail-project.component.html

```diff
@@ -1,122 +1,168 @@
-            } @else {
-            <span class="float-end budget">of {{ year.budget | currencyFormat }}</span>
-            }
-          </div>
-          <hr class="solid" />
-          <div class="clearfix">
-            <span class="float-start"><fa-icon [icon]="faArrowUp" transform="rotate-45" style="color: #c0392b"></fa-icon> Money Out</span>
-            <span class="float-end">-{{ year.spend | currencyFormat }}</span>
-          </div>
-          <div class="clearfix mt-3">
-            <span class="float-start"><fa-icon [icon]="faArrowDown" transform="rotate-45" style="color: #2ecc71"></fa-icon> Money In</span>
-            <span class="float-end">{{ year.earned | currencyFormat }}</span>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-12">
+          <div class="wrap">
+            <div class="row align-items-center">
+              <div class="col text-center mt-3 mb-3 month">
+                <h2>Latest transactions</h2>
+              </div>
+            </div>
+    
+            <div class="table-responsive">
+              <table mat-table [dataSource]="transactions$" class="w-100">
+                <ng-container matColumnDef="date">
+                  <th mat-header-cell *matHeaderCellDef class="ps-4"> Date </th>
+                  <td mat-cell *matCellDef="let row" class="ps-4">
+                    <div class="text-secondary">{{row.date | date: 'dd/MM/yyyy'}}</div>
+                  </td>
+                </ng-container>
+      
+                <ng-container matColumnDef="description">
+                  <th mat-header-cell *matHeaderCellDef> Description </th>
+                  <td mat-cell *matCellDef="let row">
+                    <div class="d-flex align-items-center py-3">
+                      <div class="transaction-icon me-3">
+                        <fa-icon 
+                          [icon]="row.type === 'Income' ? faArrowDown : faArrowUp"
+                          transform="rotate-45"
+                          [style.color]="row.type === 'Income' ? '#2ecc71' : '#c0392b'"
+                          size="lg">
+                        </fa-icon>
+                      </div>
+                      <div>
+                        <div class="fw-medium">{{row.description}}</div>
+                        <div class="text-secondary small">{{row.type === 'Income' ? 'Income' : 'Expense'}}</div>
+                      </div>
+                    </div>
+                  </td>
+                </ng-container>
+
+                <ng-container matColumnDef="amount">
+                  <th mat-header-cell *matHeaderCellDef class="pe-4 text-end"> Amount </th>
+                  <td mat-cell *matCellDef="let row" class="pe-4 text-end">
+                    <div [class]="row.type === 'Income' ? 'text-success fw-medium' : 'text-danger fw-medium'">
+                      {{row.type === 'Income' ? '+' : '-'}}${{row.amount}}
```

Comment:
```suggestion
                      {{ row.amount | currencyFormat }}
```
Here I suggest you just use the currencyFormat pype, because we can have a lot of type of currencies and we can't use just '$', this pype will format the currency correctly with languague and currency of the user, and about the symbol to minus or plus, the best approach is to change it in the backend instead here.